CHANGELOG.md

```diff
@@ -2,8 +2,12 @@
 
 ## not released yet
 
+#### Features
+- Added `--ignore-glacier-warnings` option to `cp` and `select` commands. ([#346](https://github.com/peak/s5cmd/issues/346))
```

Comment:
```suggestion
- Added `--ignore-glacier-warnings` option to `cp`, `mv`, and `select` commands. ([#346](https://github.com/peak/s5cmd/issues/346))
```

Reply:
Also this PR adds `force-glacier transfer` flag to the `select` command.

Reply:
done